src/Modules/OrchardCore.Commerce.Payment/ViewModels/PaidStatus.cs

```diff
@@ -0,0 +1,14 @@
+using System.Text.Json.Serialization;
+
+namespace OrchardCore.Commerce.Payment.ViewModels;
+
+[JsonConverter(typeof(JsonStringEnumConverter))]
+public enum PaidStatus
+{
+    Succeeded = 0,
+    WaitingPayment = 1,
+    WaitingStripe = 2,
+    NotFound = 3,
+    Failed = 4,
+    NotThingToDo = 5,
```

Comment:
it is for operation , please see the PaymentBaseController.
maybe here the name of status is not correct, not paidstatus, not paymentstatus. I will change it to PaymentOperationStatus.
NothingTodo is diff. from Succeeded , because it transfer order to front end by using a third library in the original code.